web/src/store/index.ts

```diff
@@ -39,6 +39,9 @@ export interface State {
     selectionMode: boolean;
     subimageSelection: number[] | null;
     jobTypes: JobDetail[];
+    tours: Tour[];
+    selectedTour: Tour[];
```

Comment:
Changed in daf78f249df149db85dfc37b41c4e39340088e96

Reply:
Thanks. I see now that the `expanded` prop for the vuetify grid expects an array. I still think that `selectedTour` in the store should just be a `Tour` object, in case other components end up caring about it.

Reply:
Agreed. If some API requires a list of objects, and you want to maintain a single one, that sounds like a nice application for a computed property that yields a singleton list.